src/main/cpp/t03_points.cpp

```diff
@@ -23,10 +23,33 @@
 
 #include "t03_points.h"
 #include <iostream>
-
-
+#include<set>
+#include<map>
+#include<vector>
+#include<cmath>
 using namespace std;
 
 int t03_points() {
-    
+vector <int> a;
+multiset <double> dist;
+map <double,vector<int> > points;
+int n, x, y;
+double d;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The scope of the variable 'd' can be reduced.](https://app.codacy.com/app/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4562198)